src/project/projectModule.js

```diff
@@ -0,0 +1,63 @@
+import { projectFactory } from './projectFactory';
+// import { CreateDomElement } from '../helpers/createDomElements';
+// import { ConfigurePage } from '../helpers/configurePage';
```

Comment:
Kindly remove comments. They are not allowed.

Reply:
thank this was changed